ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/http-client/naming/main/module.xml

```diff
@@ -15,6 +15,7 @@
         <module name="java.naming"/>
         <module name="org.wildfly.http-client.common"/>
         <module name="org.wildfly.client.config"/>
+        <module name="org.jboss.ejb-client" />
```

Comment:
@ropalka does this mean there will also need to be a change to dependencies on clients using WildFly HTTP Client?

Reply:
No when such clients use it either via maven dependency or JBoss Modules dependency mechanism @darranl .

Reply:
On the clients though are we sure they will have the required dependency, they could use a strategy like we do ourselves and exclude transitive dependencies?

Reply:
If ejb-client will not be on the classpath they will get ClassNotFoundException and will need to include it @darranl